include/quiche.h

```diff
@@ -38,7 +38,7 @@ extern "C" {
 //
 
 // The current QUIC wire version.
-#define QUICHE_PROTOCOL_VERSION 0xff000017
+#define QUICHE_PROTOCOL_VERSION 0xff000018
```

Comment:
I think we want to expose "old" in `quiche.h`

Reply:
The Rust value is also private. I think we shouldn't encourage people to use that version directly so leaving it as private seems a better option.

Reply:
fine by me